featuretools/synthesis/get_valid_primitives.py

```diff
@@ -9,7 +9,11 @@
 
 
 def get_valid_primitives(
-    entityset, target_dataframe_name, max_depth=2, selected_primitives=None
+    entityset,
+    target_dataframe_name,
+    max_depth=2,
+    selected_primitives=None,
+    **dfs_kwargs,
```

Comment:
Would `**kwargs` be more recognizable?  We could rely on the docstring to specify valid keywords

Reply:
For sure. I wasn't sure if it was good to specify in the art name where the keyword args would get used, but I'm happy to make it more recognizable.

Reply:
After more time to consider I like the specificity of `**dfs_kwargs`

Reply:
Sounds good. If you saw the push with this change before I saw this comment---no you didnt!